config.frontend.test.yaml

```diff
@@ -98,6 +98,19 @@ wiktionary_project: &wiktionary_project
                 options: *default_options
           /{api:sys}: *default_sys
 
+commons_project: &commons_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - path: projects/v1/commons.wmf.yaml
```

Comment:
The point I've had on the meeting about being able not to create a commons project. We can just use the default project and add the recommned-captions right here in the config